Make lsb_release pick Gobo information. Suggested by Wilson Moura.

diff -Nur lsb-release-1.4.old/lsb_release lsb-release-1.4/lsb_release
--- lsb-release-1.4.old/lsb_release	2000-12-15 05:19:16.000000000 -0200
+++ lsb-release-1.4/lsb_release	2017-02-19 15:00:03.991988234 -0300
@@ -45,8 +45,8 @@
 INFO_ROOT="/etc"                              # directory of config files
 INFO_LSB_FILE="lsb-release"                   # where to get LSB version
 INFO_DISTRIB_SUFFIX="release"                 # <distrib>-<suffix>
-ALTERNATE_DISTRIB_FILE="/etc/debian_version"  # for Debian [based distrib]
-ALTERNATE_DISTRIB_NAME="Debian"               #     "
+ALTERNATE_DISTRIB_FILE="/etc/GoboLinuxVersion"  # for Debian [based distrib]
+ALTERNATE_DISTRIB_NAME="GoboLinux"               #     "
 CHECKFIRST="/etc/redhat-release"              # check it before file search
 
 # Defines our exit codes
